GIGZ-BACK/prisma/schema.prisma

```diff
@@ -7,8 +7,105 @@ datasource db {
+  @@index([genre_id], map: "genre_id")
+}
+
+model artist {
+  id               Int     @id @default(autoincrement())
+  name             String  @db.VarChar(255)
+  description      String? @db.LongText
+  spotify_link     String? @db.VarChar(255)
+  instagram_link   String? @db.VarChar(255)
+  facebook_link    String? @db.VarChar(255)
+  soundcloud_link  String? @db.VarChar(255)
+  youtube_link     String? @db.VarChar(255)
+  apple_music_link String? @db.VarChar(255)
+  website_link     String? @db.VarChar(255)
+  deezer_link      String? @db.VarChar(255)
+}
+
+model capacity {
+  id       Int    @id @default(autoincrement())
+  max      Int
+  color    String @db.VarChar(7)
+  bg_color String @db.VarChar(7)
+  host     host[]
+}
+
+model genre {
+  id            Int             @id @default(autoincrement())
+  name          String          @db.VarChar(255)
+  account_genre account_genre[]
+}
+
+model host {
+  id             Int       @id @default(autoincrement())
+  name           String    @db.VarChar(255)
+  description    String?   @db.LongText
+  website_link   String?   @db.VarChar(255)
+  facebook_link  String?   @db.VarChar(255)
+  instagram_link String?   @db.VarChar(255)
+  capacity_id    Int
+  host_type_id   Int
+  capacity       capacity  @relation(fields: [capacity_id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "host_ibfk_1")
+  host_type      host_type @relation(fields: [host_type_id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "host_ibfk_2")
+
+  @@index([capacity_id], map: "capacity_id")
+  @@index([host_type_id], map: "host_type_id")
+}
+
+model host_type {
+  id       Int    @id @default(autoincrement())
+  name     String @db.VarChar(255)
+  color    String @db.VarChar(7)
+  bg_color String @db.VarChar(7)
+  host     host[]
+}
+
+model liked_account {
+  id                                           Int     @id @default(autoincrement())
+  liker_account                                Int
+  liked_account                                Int
+  account_liked_account_liker_accountToaccount account @relation("liked_account_liker_accountToaccount", fields: [liker_account], references: [id], onDelete: Cascade, map: "liked_account_ibfk_1")
+  account_liked_account_liked_accountToaccount account @relation("liked_account_liked_accountToaccount", fields: [liked_account], references: [id], onDelete: Cascade, map: "liked_account_ibfk_2")
+
+  @@unique([liker_account, liked_account], map: "liker_account_liked_account")
+  @@index([liked_account], map: "liked_account")
+}
+
+model settings {
+  id             Int                @id @default(autoincrement())
+  invisible_mode Int                @default(0) @db.TinyInt
+  dark_mode      settings_dark_mode @default(disabled)
+  account_id     Int
+  account        account            @relation(fields: [account_id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "settings_ibfk_1")
+
+  @@index([account_id], map: "account_id")
+}
```

Comment:
Not good the invisble_mode, available would have been better, but bref....

Reply:
But bref...